src/main.rs

```diff
@@ -191,5 +244,14 @@ async fn basic_test() {
 
     // Start the backend and wait for SIGINT signal.
     d_web_backend.start().await.expect("Unable to start");
+    d_web_backend.create_group().await.expect("Unable to create group");
+
+    // Set and get group name
+    let group_key = d_web_backend.groups.keys().next().cloned().expect("Group not found");
+    let group = d_web_backend.get_group(group_key.clone()).await.expect("Group not found");
+    group.set_name("Test Group").await.expect("Unable to set group name");
```

Comment:
Can we get these text strings out into constants instead? Might be good to double check what the rust conventions for this are.

Reply:
Good point!

13d7e85607e54494915ce1595f740a30357e58da

Does this look good to you?

Reply:
Will move these constants out of the test and at the top of the script